examples/python/aoa.py

```diff
@@ -0,0 +1,97 @@
+# Originally developed by JSBSim Team
+# Modified by Guilherme A. L. da Silva - aerothermalsolutions.co
+# Calculation required by aircraft icing enginering
+
+import jsbsim
+import xml.etree.ElementTree as ET
+import matplotlib.pyplot as plt
+import matplotlib.style as style
+style.use('fivethirtyeight')
+
+# function to change CG in aircraft xml
+# change the directory to the aircraft to be studied
+def changeCG(cgPos,readOnly):
+    tree = ET.parse('../../aircraft/global5000/global5000.xml')
+    root = tree.getroot()
+
+    for x in root.findall('mass_balance/location'):
+        cg = x.find('x').text
+        if not readOnly:
+             x.find('x').text=str(cgPos)
+             tree.write('../../aircraft/global5000/global5000.xml')
+    return cg
+
+#Fuel Max for Global5000
+fuelmax=8097.63
+
+#prepare subplots to overlay plots
+fig, ax = plt.subplots(figsize=(10,8))
+
+#Define here the payloads to be studied
+payload=[1500,15172/2,15172]
+#Define here the mass of fuel
+fuel=[1000,fuelmax/2,fuelmax]
+#three cases for weight
+weight=["light","mid","heavy"]
+
+#get the original CG from aircraft xml
+cgOrig=float(changeCG(0,True))
+
+#vary CG in the study
+cgPos=[cgOrig*0.95,cgOrig*1.05]
+
+#vary altitude
+h_ft=[8000,30000]
+
+#run the simulation varying CG, altitude, speed and total weight
+for j in range(2):
+    cg=changeCG(cgPos[j],False)
+    fdm = jsbsim.FGFDMExec('../../') 
```

Comment:
```suggestion
    fdm = jsbsim.FGFDMExec(PATH_TO_JSBSIM_FILES) 
    cg=changeCG(fdm, cgPos[j],False)
```
I suggest to switch the order of these 2 lines so that `changeCG` can make reference to the FDM root directory rather than using an hardcoded relative path.

Also I'd suggest to use a global variable `PATH_TO_JSBSIM_FILES` that is defined at the very top of the script and documented as being needed to be updated if the script is copied in another directory.